### PR TITLE
Update BCM SAI version to 7.1.0.0-2

### PR DESCRIPTION
**Pull request by judyjoseph**
#### Why I did it

Contains broadcom fixes in CSP needed especially for macsec

```
CS00012242614                [SAI 7.0, macsec] MACSEC MKA SAK Rekey fails
CS00012245841                [SAI 7.1, macsec, J2C+] MKA session not established when macsec profile attached again to interfaces in Portchannel
CS00012245837                [SAI 7.1, macsec, J2C+] syncd/orchagent exit, when macsec configured on Portchannel members -- with no macsec config on peer interfaces.
CS00012245790                [SAI 7.1, BCM88850 (Jericho2c+)] SAK Rekey fails to establish mka session , 128, 256, XPN – On regular interface, portchannel
CS00012245570                [SAI 7.1, J2C+] SAI trace and syncd exit with mismatch in local/remote cipher suites 
CS00012245349                [SONiC J2c+ Macsec]: Syncd crashes in _brcm_sai_dnx_macsec_cfg_egr_data_objects when a macsec session link flaps
CS00012248081                SAI_API_MACSEC:brcm_sai_dnx_set_macsec_sc_attribute:1784 sc 0 attrib 9 failed with error 0xfffb0000.
CS00012248082                segmentation fault in SAI macsec code
CS00012248144                SONiC J2C+ Macsec: IPv6 Neighbor Solicitation packets are missing an Ethernet header
CS00012248516                SONiC J2c+ Macsec: Interfaces with 256-bit cipher configured will only program the lower 128-bits of the SAK after link flap
```

```
CS00012235031 | SAI_API_NEXT_HOP_GROUP:brcm_sai_dnx_create_next_hop_group:1956   Unable to reserve ECMP fec block failed with error -4
```

#### How I did it
Tried it with macsec on a multi-asic chassis linecard.

#### How to verify it
Tried it with macsec on a multi-asic chassis linecard.

Validated with fib/bgp tests and interfaces up in Jr2/J2C+ devices 
XGS based devices the interfaces are up. No change expected as the SAI bcm debian remains same for XGS 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

File: platform/broadcom/sai.mk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-LIBSAIBCM_VERSION = 7.1.0.0-1
+LIBSAIBCM_VERSION = 7.1.0.0-2
```
Comment (lguohan) [inline platform/broadcom/sai.mk:1]: can you only update the dnx platform?

Comment (judyjoseph) [reply platform/broadcom/sai.mk]: @lguohan have updated the saibcm only for DNX

**Comment by judyjoseph** [comment]
Close this PR and open a new one to update the SAI to new one for DNX without macsec.

**Pull request closed without merging**